doc: nrf: releases: Clarify supplicant switch to kernel heap (#20108)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

       For example, ``CONFIG_WPA_SUPP=y`` to ``CONFIG_WIFI_NM_WPA_SUPPLICANT=y``.
       Update your application configurations to use the new namespace.

>      * Please note that you need to reconcile application heap and kernel heap usage appropriately to accommodate this switch from application to kernel heap.

Reviewer: ```suggestion
     * You need to reconcile the application heap and kernel heap usage appropriately to accommodate this switch from application to kernel heap.
```
